08/calculator.go

```diff
@@ -0,0 +1,5 @@
+package calculator
+
+func Add(a int, b int) int {
```

Comment:
Just so you know, when you have a series of parameters of the same type, you can declare them like this:

```suggestion
func Add(a, b int) int {
```

Reply:
Thank you for the suggestion.